backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/config/JwtRequestFilter.java

```diff
@@ -0,0 +1,59 @@
+package com.riptFitness.Ript_Fitness_Backend.config;
+
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
```

Comment:
Do you need an annotation here so dependency injection automatically calls your constructor? I see you put them in the others but didn't know if this one needed an @ annotation as well